packages/puppeteer-core/src/common/bidi/Browser.ts

```diff
@@ -192,6 +230,14 @@ export class Browser extends BrowserBase {
       'browsingContext.contextCreated',
       this.#onContextCreated
     );
+    this.#connection.off(
+      'browsingContext.fragmentNavigated',
```

Comment:
Style question wouldn't a loop here and on subscription be better that way we add a map like the reset of the classes in BiDi?